WordPress/WordPressTest/FeatureFlagTests.swift

```diff
@@ -0,0 +1,33 @@
+import XCTest
+@testable import WordPress
```

Comment:
Neat!

I am not sure if this link is still accurate, but I always make testing cases final, in case that makes tests run faster, even if it is just a tad.

https://developer.apple.com/swift/blog/?id=27

Reply:
Thanks for the link. I see no reason for the class _not_ to be `final`, so I've pushed an additional commit that applies this designation.